### PR TITLE
222 remove board entity

### PR DESCRIPTION
**Pull request by ihor-il**
Removed Board entity and moved everything board-related to project service/controller

### CODE REVIEW

**Comment by Anatolii-Ivanov** [comment]
Can you, at least partially, update the scheme of db in readme file?
Everything else looks fine

**Comment by ihor-il** [comment]
> Can you, at least partially, update the scheme of db in readme file? Everything else looks fine

I used [this](https://github.com/KarnerTh/mermerd) tool to generate ERD. We should test it and if it works fine we could use it everywhere or even try to integrate it into CI/CD

File: backend/Tasque.BLL/Services/ProjectService.cs [hunk 1]
```diff
@@ -177,4 +164,29 @@ public async Task ChangeUserRole(ChangeUserRoleDto roleDto)
...
+    }
+
+    public async Task<BoardInfoDto> UpdateBoard(BoardInfoDto board)
+    {
+        var proj = _db.Projects
+            .Include(x => x.Columns).ThenInclude(x => x.Tasks)
+            .Include(x => x.Users)
+            .FirstOrDefault(x => x.Id == board.Id)
+            ?? throw new CustomNotFoundException("project board");
+        var mapped = _mapper.Map(board, proj);
```
Comment (oleksandr-vovk) [inline backend/Tasque.BLL/Services/ProjectService.cs]: mapping exception
This row throws an exception on moving task card.

File: backend/Tasque.BLL/Services/ProjectService.cs [hunk 2]
```diff
@@ -177,4 +164,29 @@ public async Task ChangeUserRole(ChangeUserRoleDto roleDto)
...
+        _db.BoardColumns.AddRange(proj.Columns.Where(x => x.Id == 0));
+
```
Comment (oleksandr-vovk) [inline backend/Tasque.BLL/Services/ProjectService.cs:307]: Did you forget to save task position changes?

**Review by oleksandr-vovk — CHANGES_REQUESTED** [review]

File: backend/Tasque.BLL/Services/ProjectService.cs (hunk 2 above)
Comment (ihor-il) [reply backend/Tasque.BLL/Services/ProjectService.cs:307]: do we have task position?

Comment (oleksandr-vovk) [reply backend/Tasque.BLL/Services/ProjectService.cs:307]: task has a columnId property and it changes if we move the task on the board

**Review by oleksandr-vovk — APPROVED** [review]
Works well

**Review by andpii — APPROVED** [review]

File: frontend/src/app/components/tasque-board/tasque-board.component.ts [hunk 3]
```diff
@@ -39,30 +44,35 @@ export class TasqueBoardComponent implements OnInit, OnDestroy {
...
-    color: 'green',
-    title: 'Story'
-  },
-];
-
-  constructor(formBuilder: FormBuilder,
-    private route: ActivatedRoute, 
-    private boardService: BoardService, 
+  public issueTypes: TasqueDropdownOption[] = [
+    // TODO: Here will be all project issue types
```
Comment (DaniilPoiarkov) [inline frontend/src/app/components/tasque-board/tasque-board.component.ts:48]: Pls, remove this hardcoded data

Comment (goose0chka) [reply frontend/src/app/components/tasque-board/tasque-board.component.ts:48]: this is out of the scope of this task

**Pull request merged**